apps/web/components/LanguageToggler/LanguageToggler.tsx

```diff
@@ -70,45 +71,53 @@ export const LanguageToggler = ({
 
     const responses = await Promise.all(queries)
 
+    const secondContentSlug = responses[1]?.data?.getContentSlug
+
+    // We need to have a special case for subArticles since they've got a url field instead of a slug field
+    if (secondContentSlug?.type === 'subArticle') {
+      const urls = secondContentSlug.url[otherLanguage].split('/')
+
+      if (!secondContentSlug?.title?.[otherLanguage] || urls.length < 2) {
```

Comment:
Can you add a comment on why you have the `.length < 2` check?

Reply:
Ok, will do